src/operator/core/crdHandler.go

```diff
@@ -25,12 +25,14 @@ func (dm *KVMSOperator) UpdateExternalWorkloadSecurityPolicies() {
 	dm.ExternalWorkloadSecurityPoliciesLock.Lock()
 	defer dm.ExternalWorkloadSecurityPoliciesLock.Unlock()
 
+	/* Unused code
```

Comment:
@seswarrajan , can you check if the commenting this code is ok? It is unused code anyways is my observation. I was getting lint issues hence i commented the code.

Reply:
Yes @nyrahul 
Can be removed